manala.locales/tasks/codes.yml

```diff
@@ -1,7 +1,14 @@
 ---
 
-- name: codes > Handle
+- name: Debian | codes > Handle
   locale_gen:
-    name:  "{{ (item) if (item is string) else (item.name) }}"
+    name:  "{{ (item) if (item is string) else (item.input+'.'+item.charmap) }}"
```

Comment:
What is the purpose of ```(item.input+'.'+item.charmap)``` ?

Reply:
because for redhat localedef, I need to give input and charmap separately. forgot to update README